compiler/stack.yaml

```diff
@@ -2,7 +2,15 @@
 #
 # Some commonly used options have been documented as comments in this file.
 # For advanced use and comprehensive documentation of the format, please see:
-# http://docs.haskellstack.org/en/stable/yaml_configuration/
+# https://docs.haskellstack.org/en/stable/yaml_configuration/
+
+# A warning or info to be displayed to the user on config load.
```

Comment:
I wanted to keep it during iterations because it was a reminder to make sure stack did the right thing. I had some complications before updating my stack.